spec/chromium-spec.js

```diff
@@ -223,6 +223,26 @@ describe('chromium feature', () => {
       b = window.open(`file://${fixtures}/pages/window-open-size.html`, '', 'show=no')
     })
 
+    for (const show in [true, false]) {
+      it(`inherits parent visibility over parent {show=${show}} option`, (done) => {
+        let w = new BrowserWindow({show: show})
```

Comment:
```js
const w = new BrowserWindow({show})

Reply:
technically the `let` shouldn't be here, but `w` is declared slightly higher as a `let` so we can't make it const here 👍

Reply:
I think you can. You just declare a new variable that simply shadows the one declared above it.